### ISSUE TITLE
Web component: Masthead L1 - Search edit combo list box - No label for this control. 

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#4773]
## Detailed description
On JAWS FF/Voiceover iPhone

Issue: Search edit combo list box - No label for this control. 

Expected: give label as "search"

**Comment by ariellalgilmore** [comment]
@Praveen-sr can you re-test this? Looks like this is working now.

**Comment by Praveen-sr** [comment]
@ariellalgilmore Re tested this on JAWS FF/Voiceover iPhone, Not fixed.

Placeholder is provided instead of a label. Place holder cannot be considered as an replacement for an Accessible Name. And also Because it is not rendered reliably across the different versions of Screen readers and Browsers. We recommend using aria-label here. 
Please refer the following link on the same. 
https://www.w3.org/WAI/tutorials/forms/instructions/
